src/test/arbitrum/AaveV3ArbWSTETHListingPayloadTest.t.sol

```diff
@@ -0,0 +1,77 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.17;
+
+import 'forge-std/Test.sol';
+import {AaveV3Arbitrum} from 'aave-address-book/AaveAddressBook.sol';
+import {ProtocolV3TestBase, InterestStrategyValues, ReserveConfig} from 'aave-helpers/ProtocolV3TestBase.sol';
+import {AaveGovernanceV2} from 'aave-address-book/AaveGovernanceV2.sol';
+import {TestWithExecutor} from 'aave-helpers/GovHelpers.sol';
+import {AaveV3ArbWSTETHListingPayload} from '../../contracts/arbitrum/AaveV3ArbWSTETHListingPayload.sol';
+
+contract AaveV3ArbWSTETHListingPayloadTest is ProtocolV3TestBase, TestWithExecutor {
```

Comment:
Do you think we might need some post execution tests, checking that you can supply this as collateral and also borrow against it? Does BGD recommend that?

Reply:
we have a e2e suite, but for "normal"(no custom a/s/v implementations) listing i think configuration snapshot it enough. We opt to not test the protocol standard functionality (as the protocol is tested heavily).